test/smp/regression/adp/cases/dsd_uds_1mb_3k_contexts_throughput/experiment.yaml

```diff
@@ -34,6 +34,12 @@ target:
     DD_TELEMETRY_ENABLED: "true"
     DD_PROMETHEUS_LISTEN_ADDR: tcp://127.0.0.1:5102
 
+  profiling_environment:
+    SMP_PROFILING_ENABLED: "true"
+    DD_TRACE_AGENT_URL: "unix:///var/run/datadog/apm.socket"
+    DD_PROFILING_NATIVE_PRESET: "cpu_live_heap"
+    DD_PROFILING_INLINED_FUNCTIONS: "true"
```

Comment:
Profiles are showing up named [`myservice`](https://app.datadoghq.com/profiling/explorer?query=env%3Asingle-machine-performance%20job_id%3Ae422040e-774c-46de-a405-3f6b22f61821%20experiment%3Adsd_uds_1mb_3k_contexts_throughput&agg_m=%40prof_native_cpu_cores&agg_m_source=base&agg_t=sum&extra_search_fields=%7B%22filters_query%22%3A%22%22%2C%22sample_type%22%3A%22cpu-time%22%7D&fromUser=true&selected_tf=1765570752668.0667%2C1765573261673.1216&viz=flame_graph&start=1765491577295&end=1765577977295&paused=true); setting `DD_SERVICE` will fix that.